setup.cfg

```diff
@@ -23,6 +23,7 @@ test =
     testtools
     os-testr
     mock
+    httpbin
```

Comment:
This shouldn't be here, it should be installed via tox when testing.

Reply:
Fixed.